src/userbase-js/ws.js

```diff
@@ -670,7 +677,9 @@ class Connection {
 
   async saveSeed(seedString) {
     const username = this.username
-    localData.saveSeedString(username, seedString)
+
+    if (this.rememberMe) localData.saveSeedString(username, seedString)
+
```

Comment:
We may want to put an extra warning in the Show Key modal when rememberKey is false. We can warn users that unless they copy the secret key, they won't be able to log in their account again.